packages/language-server/package.json

```diff
@@ -26,12 +26,13 @@
   "main": "./dist/index.js",
   "typings": "dist/src/index",
   "dependencies": {
-    "@prisma/get-platform": "2.2.0-dev.1",
+    "@prisma/get-platform": "2.2.0-dev.7",
```

Comment:
This is causing the `getCLIVersion()` function to get incorrect value `2.2.0-dev.7`. Trying to move it to use something else to get the correct value since this is already published. 

![image](https://user-images.githubusercontent.com/746482/85706273-a89d0580-b6ff-11ea-973f-2efa6d2e557d.png)

Reply:
but isn't 2.2.0-dev.7 correct?

Reply:
Not for the patch release, it should be `2.1.1` but LSP 2.1.2 is hardcoded to the dev version now.

![image](https://user-images.githubusercontent.com/746482/85708014-5e1c8880-b701-11ea-8cbe-7f9503eb2595.png)